client/model.go

```diff
@@ -22,13 +22,19 @@ package client
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	"fmt"
+	"net/url"
+
+	"github.com/snapcore/snapd/asserts"
 )
 
 type remodelData struct {
 	NewModel string `json:"new-model"`
 }
 
+const ErrorKindAssertionNotFound = "assertion not found"
```

Comment:
mmh, this is not a ErrorKind as we usually intend them

Reply:
I think I have reworked this as requested to match other command/client/daemon patterns.